content/sql/concepts/operators/terms/not/not.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: NOT
-Description: 'Query items that return NOT TRUE for some condition/conditions.'
+Description: 'Queries for items in a table that return NOT TRUE for some condition(s).'
```

Comment:
Should be updated like the other boolean operators to refer to expressions rather than querying tables.

Reply:
Done!